tests/isr_context_switch/main.c

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright (C) 2020 Otto-von-Guericke-Universität Magdeburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup tests
+ * @{
+ *
+ * @file
+ * @brief   Application for testing context switching triggered from IRQ
+ *
+ * @author  Marian Buschsieweke <marian.buschsieweke@ovgu.de>
+ *
+ * @}
+ */
+
+#include <stdint.h>
+#include <stdatomic.h>
+#include <stdio.h>
+
+#include "macros/units.h"
+#include "mutex.h"
+#include "periph/timer.h"
+#include "random.h"
+#include "thread.h"
+
+#ifndef TIMER
+#define TIMER               0
```

Comment:
```suggestion
#define TIMER               TIMER_DEV(0)
```

makes the code a bit nicer to read